.gitignore

```diff
@@ -119,3 +119,4 @@ migrations/*
 
 .idea/
 .idea
+andela-societies/
```

Comment:
You need to add a new line here. Why does this need to be ignored though?

Reply:
It's my virtualenv directory. So I don't think it should be added.

Reply:
Thanks.

Reply:
New line added.